frontend/tests/Utils.php

```diff
@@ -97,12 +97,32 @@ public static function gradeRun(
             "\x1f\x8b\x08\x08\xaa\x31\x34\x5c\x00\x03\x66\x6f" .
             "\x6f\x00\x03\x00\x00\x00\x00\x00\x00\x00\x00\x00"
         );
-        // An empty zip file.
+        // Creating the zip file.
+        $outputFilesContent = [
+            'easy.00.out' => 3,
+            'easy.01.out' => 5,
+            'medium.00.out' => 300,
+            'medium.01.out' => 6912,
+            'sample.out' => 3,
```

Comment:
estos deberían ser cadenas para evitar problemas de tipos.

```suggestion
            'easy.00.out' => '3',
            'easy.01.out' => '5',
            'medium.00.out' => '300',
            'medium.01.out' => '6912',
            'sample.out' => '3',
```

Reply:
Listo. Subí las correcciones en el PR #4324